message.go

```diff
@@ -95,6 +95,10 @@ func parseOtherMessage(line string) *message {
 	}
 	msg.Tags = make(map[string]string)
 
+	if len(split) >= 4 && len(split[3]) > 1 && split[3][0] == '#' {
```

Comment:
I would prefer a comment explaining why you are doing this. I know this isn't done much in the codebase but I want to start doing it more to make parsing a bit more transparent.